rq_dashboard/default_settings.py

```diff
@@ -19,3 +19,5 @@
 DEBUG = False
 RQ_DASHBOARD_WEB_BACKGROUND = "black"
 RQ_DASHBOARD_DELETE_JOBS = False
+
+RQ_DASHBOARD_JOB_SORT_ORDER = '-age'
```

Comment:
This looks a little bit confusing: we providing option to choose sorting order, but actually support only sorting by age.
I have plans to redesign the part which is responsible for fetching and displaying list of jobs, workers, etc.

Reply:
I can change the key name if you have any better suggestion, let me know.

Reply:
oh i just submitted a somewhat similar  MR ... but its just for the failed job Q ... and just reverses the current order ...  its really painful to always be jumping to the last page ... maybe its by design however to encourage people to clear out failed jobs

Reply:
Yes, It is by design to replay failed jobs in the same order.
But per-queue sorting is just unfortunate side-effect of compatibility changes.
I'll review all related patches tomorrow. And I have an idea how to handle this properly.

Reply:
Okay. Basically this has nothing to do with the configuration and It seems to be pretty rational to implement this option on frontend.

Reply:
@eoranged So, you suggest to move this option to api, may be a query_param 

```
/jobs/<queue_name>/<page>.json?job_order=[asc|desc]
```

Reply:
Yes.
Actually I almost implemented this and want to create another pull request tomorrow night.